test/e2e/run-tests.sh

```diff
@@ -21,41 +27,21 @@ if [ $# -ne 1 ]; then
     exit 1
 fi
 
+# construct and validate service directory path
 SERVICE="$1"
-
 service_test_dir="$E2E_DIR/$SERVICE"
-
 if [ ! -d "$service_test_dir" ]; then
     echo "No tests for service $SERVICE"
     exit 0
 fi
 
-# check to see if the tests use pytest
-[[ -f "$service_test_dir/__init__.py" ]] && enable_python_tests="true" || enable_python_tests="false"
-
-# find all files except under helper directory
-service_test_files=$( find "$service_test_dir" -name helper -prune -false -o -type f ! -name '.*' | sort )
+# find and execute all test scripts within the service directory
+service_test_files=$( find "$service_test_dir" -type f -name '*.sh' | sort )
 
-if [[ "$enable_python_tests" == "false" ]]; then
-  ROOT_DIR="$E2E_DIR/../.."
-  SCRIPTS_DIR="$ROOT_DIR/scripts"
-  source "$SCRIPTS_DIR/lib/common.sh"
-
-  for service_test_file in $service_test_files; do
-      test_name=$( filenoext "$service_test_file" )
-      test_start_time=$( date +%s )
-      bash $service_test_file
-      test_end_time=$( date +%s )
-      echo "$test_name took $( expr $test_end_time - $test_start_time ) second(s)"
-  done
-else
-  PYTEST_LOG_LEVEL="${PYTEST_LOG_LEVEL:-"INFO"}"
-  python bootstrap.py "${SERVICE}"
-
-  set +e
-
-  PYTHONPATH=. pytest -n auto --dist loadfile --log-cli-level "${PYTEST_LOG_LEVEL}" "${SERVICE}"
-  python cleanup.py "${SERVICE}"
-
-  set -eo pipefail
-fi
```

Comment:
hmmm. by removing the above, you are removing the ability of the contributor to easily run the python tests on their local system instead of inside the Docker python testing image (which is how $TEST_E2E_DIR/build-run-test-dockerfile.sh works).

I suppose I don't mind too much being forced to use the Docker approach to running python tests, but wondering what 
@RedbackThomson thoughts are on this?

Reply:
Yeah perhaps I don't understand why this block is being removed? It seems to just be removing the ability to run these tests locally?
I personally run tests locally because waiting for Docker and debugging inside a container is a pain.

Reply:
Yeah I thought about this a bit more, I agree keeping this would be useful. 

Since the bash tests are still useful to us (before we migrate everything to python), I found it inconvenient that `run-tests.sh` automatically ran python tests if they existed, even though the script was invoked because `enable_python_tests` was false in `kind-build-test.sh`.

I suppose there are currently 3 ways to run these tests (bash, python locally, python inside Docker), I'll try to come up with a way to allow the user to clearly specify one way.